lessons/day1/readme.md

```diff
@@ -1,8 +1,10 @@
-_Jumpstart Live (JSL)_
+_JumpStart Live (JSL)_
 # Day 1
 
 ## Prep work
-* Review your [Madlibs](https://github.com/Ada-Developers-Academy/jump-start/tree/master/lessons/grammar) solution and be prepared to talk with others about your code
+* Review your [Madlibs](https://github.com/Ada-Developers-Academy/jump-start/tree/master/lessons/grammar) solution and be prepared to talk with others about your code.
+* Watch [Day 1 Videos](https://adaacademy.hosted.panopto.com/Panopto/Pages/Sessions/List.aspx?folderID=66c9e5fc-8a11-4db2-b0fe-ebc516d45071) and review [lesson notes](#lessons).
```

Comment:
Please make the review the lesson notes its own bullet point so students don't miss it

Reply:
Makes sense. Done.